### PR TITLE
translate(rota): setup translation and translate to PT

### PR DESCRIPTION
**Pull request by leocaseiro**
#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

**Review by vdfdev — APPROVED** [review]

**Comment by vdfdev** [comment]
CI is failing

**Comment by leocaseiro** [comment]
> CI is failing

Of course I forgot to fix the tests 😫

File: web/src/games/fourinarow/locales/pt.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Quatro em linha",
   "description": "Parecido com Conecta 4",
-  "descriptionTag": "Jogue Quatro em linha de graça\n\nVocê pode jogar no modo de multi-jogador contra um amigo online, ou compartilhar seu dispositivo e jogar localmente contra um amigo.\n",
+  "descriptionTag": "Jogue Quatro em linha de graça\n\nVocê pode jogar no modo de multi-jogador contra um amigo online, ou compartilhar seu dispositivo e jogar localmente contra um amigo.",
```
Comment (leocaseiro) [inline web/src/games/fourinarow/locales/pt.json:4]: no need to break line here

File: web/src/games/rota/locales/pt.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "player_turn": "Vez de {{ pName }}",
+    "game_over": {
+      "you_won": "você ganhou",
+      "you_lost": "você perdeu",
+      "player_won": "{{ pName }} ganhou"
+    }
+  },
+
+  "local": {
+    "blue": "Blue",
```
Comment (leocaseiro) [inline web/src/games/rota/locales/pt.json:23]: Ooops!

**Review by vdfdev — APPROVED** [review]

**Pull request merged**